exec-healthz/exechealthz.go

```diff
@@ -37,15 +39,47 @@ import (
 // 3. 404 for unknown endpoints
 
 var (
-	port       = flag.Int("port", 8080, "Port number to serve /healthz.")
-	cmd        = flag.String("cmd", "echo healthz", "Command to run in response to a GET on /healthz. If the given command exits with 0, /healthz will respond with a 200.")
+	cmdDefault = "echo healthz"
```

Comment:
having a default cmd is not that useful.  I won't block this PR on it, but if anything else needs to be fixed, I would just remove this and demand at least one --cmd